flink-scala/pom.xml

```diff
@@ -272,6 +272,12 @@ under the License.
 						<goals>
 							<goal>test-jar</goal>
 						</goals>
+						<configuration>
+							<excludes>
+								<!-- test-jar is still used by JUnit4 modules -->
+								<exclude>META-INF/services/org.junit.jupiter.api.extension.Extension</exclude>
```

Comment:
Maybe highlight this with a TODO to remove this exclude after the migration is complete and JUnit4 is not necessary anymore?